error({ a = 1 })
return true
